xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -134,6 +136,11 @@ enum AppAction: CustomLogStringConvertible {
     /// Set settings sheet presented?
     case presentSettingsSheet(_ isPresented: Bool)
 
+    /// Set recovery phrase on recovery phrase component
+    static func setRecoveryPhrase(_ phrase: String) -> AppAction {
```

Comment:
This is a cheap trick for exposing child domain actions in parent context. It looks like a parent enum case, but it really is just a factory for a boxed child action.